src/LevelEditor.cpp

```diff
@@ -1461,6 +1461,7 @@ class LevelEditorSelectionPopup{
 			if(parent!=NULL){
                 //draw name
 				TexturePtr& tex = scenery ? (parent->getCachedTextTexture(renderer, scenery->sceneryName_.empty()
+					/// TRANSLATORS: Texture name

```

Comment:
I think this comment is wrong. Firstly, the `getCachedTextTexture` returns a texture with specified string, e.g. if you input "Hello World" it will return a texture containing "Hello World". Secondly, there is actually a scenery block named "Custom scenery block", just scroll to the last item in the toolbox when adding item in the scenery layer.

Reply:
Call it "Block name" then?

Reply:
I am not allowed to reply to your comment above, so I'm doing it here.

>>    TRANSLATORS: Context: Undo/Redo ...

> I think this comment may be a little bit vague, I prefer something like "This is the description of a level editing command which is used in Undo/Redo support".

For translators, that is even more vague. Translators need to know where the string is in the sentence. How about "Replaces %s in "Undo %s" / "Redo %s" in the editor"?